distribution/rpm/src/main/packaging/init.d/elasticsearch

```diff
@@ -32,19 +32,19 @@ if [ -f /etc/rc.d/init.d/functions ]; then
 fi
 
 # Sets the default values for elasticsearch variables used in this script
-ES_USER="${packaging.elasticsearch.user}"
-ES_GROUP="${packaging.elasticsearch.group}"
-ES_HOME="${packaging.elasticsearch.home.dir}"
-MAX_OPEN_FILES=${packaging.os.max.open.files}
-MAX_MAP_COUNT=${packaging.os.max.map.count}
-LOG_DIR="${packaging.elasticsearch.log.dir}"
-DATA_DIR="${packaging.elasticsearch.data.dir}"
-CONF_DIR="${packaging.elasticsearch.conf.dir}"
+ES_USER="elasticsearch"
+ES_GROUP="elasticsearch"
+ES_HOME="/usr/share/elasticsearch"
```

Comment:
Thank you for removing templating of these properties! It makes these things much more readable.